### PR TITLE
Fix ssue #692: Update code in filer/tests/spec/time-flags.spec.js

### PR DESCRIPTION
**Pull request by Violet-XiaoWeiHuang**
"var" was replaced by "let", as the passed variable may be modified later. In this case, I avoid using "const".

### CODE REVIEW

**Comment by rscotchmer** [comment]
Hi!  It looks like you're off to a great start, but I think there are some changes to be made yet.  [On line 13](https://github.com/filerjs/filer/blob/31fd258722ffe63075649dc062414f60ead59171/tests/spec/time-flags.spec.js#L13), you've got a comma instead of a semi-colon, which I think is why your Travis CI build is failing -- I copied your code into my local version of filer, and was able to run the tests perfectly after I fixed that.

You're also missing 'use strict' at the top of your file, which you need to use so the file runs in strict mode.  I think you could also change the 3 "lets" on lines 3, 4, and 5 to "const"; the values there should never change, so it's safe to use const.

**Comment by Violet-XiaoWeiHuang** [comment]
Thank you for your comment. It's strange I tried locally, all tests passed. 
And [in line 13](https://github.com/filerjs/filer/blob/31fd258722ffe63075649dc062414f60ead59171/tests/spec/time-flags.spec.js#L13) you mentions the comma should be correct as name is one of the data member.  @rscotchmer

**Comment by rscotchmer** [comment]
I feel you -- I had all tests passing locally, and then when I put up the pull request it decided to fail a few -- the joys of programming, eh?  I'm glad all the tests are passing for you now, though!

You're definitely right about that comma!  My bad, I should've taken a closer look at that one.

**Comment by Violet-XiaoWeiHuang** [comment]
Thank you @ithompson4.

File: tests/spec/time-flags.spec.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+
+const Filer = require('../../src');
+const util = require('../lib/test-utils.js');
+const expect = require('chai').expect;
 
 describe('node times (atime, mtime, ctime) with mount flags', function() {
 
-  var dirname = '/dir';
-  var filename = '/dir/file';
+  let dirname = '/dir';
```
Comment (humphd) [inline tests/spec/time-flags.spec.js]: All these `let` keywords can be switched to `const`, since the variables aren't going to change value.

**Pull request merged**